tools/buildsys/src/builder.rs

```diff
@@ -11,10 +11,13 @@ use duct::cmd;
 use nonzero_ext::nonzero;
 use rand::Rng;
 use sha2::{Digest, Sha512};
-use snafu::{ensure, ResultExt};
+use snafu::{ensure, OptionExt, ResultExt};
 use std::env;
+use std::fs;
```

Comment:
```suggestion
use std::fs::{File, self};
```

nit: `File` is usually used without the module prefix since it's so common.